category.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PHPMD\Category;
+
+class Category
+{
+    public static $checks;
+}
+
+Category::$checks = array(
```

Comment:
Strange. Clearly, I don't know PHP, but I don't see why defining `$checks` within `Category` would be any different then it was within `JSONRenderer`?

Just thinking in an OOP-way, I'd write something like:

```php
class Category
{
    private static $checks = array(
        # ...
    );

    public function getCategory(checkName)
    {
        return $this->$checks[checkName][0];
    }

    public function getRemeditionPoints(checkName)
    {
        return $this->$checks[checkName][1];
    }
}
```

Sidenote: do we have enough test coverage in this project that it'll at least catch syntax errors?